<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>glxtension</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="rendererValue">
        <property name="text">
         <string>RendererValue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="versionValue">
        <property name="text">
         <string>VersionValue</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="vendorValue">
        <property name="text">
         <string>VendorValue</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="glslVersionValue">
        <property name="text">
         <string>GLSLVersionValue</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="glWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="glLogoLabel">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../resources.qrc">:/opengl/opengl-logo</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="glslVersionLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Shading Language Version:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="versionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>OpenGL Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rendererLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Renderer:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="vendorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vendor:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="extensionTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="extensionTreeTab">
       <attribute name="title">
        <string>Extension Tree</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="extensionsTree">
          <property name="toolTip">
           <string>Double click to open the extension's specification.</string>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Corporation / Institution</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QToolButton" name="collapseButton">
            <property name="toolTip">
             <string>Collapse all corporations</string>
            </property>
            <property name="text">
             <string>Collapse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="expandButton">
            <property name="toolTip">
             <string>Expand all corporations</string>
            </property>
            <property name="text">
             <string>Expand</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="filterLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Filter:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="buddy">
               <cstring>filterText</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="filterText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Type here to start searching for extensions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="clearFilterButton">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="extensionSpecTab">
       <attribute name="title">
        <string>Read the Specification</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="extensionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Extension:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="extensionComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWebView" name="extensionSpecView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterText</tabstop>
  <tabstop>extensionsTree</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>collapseButton</sender>
   <signal>clicked()</signal>
   <receiver>extensionsTree</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expandButton</sender>
   <signal>clicked()</signal>
   <receiver>extensionsTree</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>extensionsTree</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearFilterButton</sender>
   <signal>clicked()</signal>
   <receiver>filterText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
